src/interfaces/widgets/module.py

```diff
@@ -0,0 +1,300 @@
+"""
+
+    This module generate serveral combination frame based object.
+
+"""
+
+from PyQt5.QtWidgets import QWidget
+from PyQt5.QtCore import Qt
+
+from .button import Button, RadioButton
+from .frame import Frame
+from .groupbox import GroupBox
+from .label import Label
+from .layout import HorizontalLayout
+from .lineedit import LineEdit
+from .spacer import HorizontalSpacer
+
+
+class BaseTwoLabelFrame(Frame):
+
+    label_one : Label = None
+    label_two : Label = None
+
+    def __init__(self, widget, **kwargs):
+        super(BaseTwoLabelFrame, self).__init__(widget, **kwargs)
+
+        kwargs.pop('name', None)
+        get_param = lambda x : kwargs.get(x)
+
+        label_one_name = get_param("label_one_name")
+        label_two_name = get_param("label_two_name")
+
+        label_one_text = get_param("label_one_text")
+        label_two_text = get_param("label_two_text")
+
+        layout = HorizontalLayout(self)
+
+        self.label_one = Label(self, name=label_one_name, **kwargs)
+        self.set_label_one_text(label_one_text)
+        layout.addWidget(self.label_one)
+
+        spacer = HorizontalSpacer()
+        layout.addItem(spacer)
+
+        self.label_two = Label(self, name=label_two_name, **kwargs)
+        self.set_label_two_text(label_two_text)
+        layout.addWidget(self.label_two)
+
+    def set_label_one_text(self, text:str):
+        self.label_one.setText(text)
+
+    def set_label_two_text(self, text:str):
+        self.label_two.setText(text)
+
+    def get_label_one(self):
+        return self.label_one
+
+    def get_label_two(self):
+        return self.label_two
+
+
+class SectionTitleFrame(BaseTwoLabelFrame):
+
+    def __init__(self, widget, **kwargs):
+        super(SectionTitleFrame, self).__init__(widget,
+                                                name="section_title_frame",
+                                                label_one_name="section_title",
+                                                label_two_name="section_hint",
+                                                align=Qt.AlignVCenter, **kwargs)
+
+
+class ConfigFrame(BaseTwoLabelFrame):
+
+    def __init__(self, widget, **kwargs):
+        super(ConfigFrame, self).__init__(widget, name="config_frame",
+                                          align=Qt.AlignVCenter, **kwargs)
+
+    def setTitle(self, text:str):
+        super().set_label_one_text(text)
```

Comment:
You shouldn't need to call `super()` here since this is a subclass making the function automatically exposed.

Reply:
`set_label_one_text` is a public function that we want outer scope calling. When people want to set the text for the first label within the frame, they can call `set_label_one_text`. The reason why we have a `setTitle` here is I want `ConfigFrame` to have a wrapper function with a "easy-understanding" name.